.changes/unreleased/FEATURES-20230912-123837.yaml

```
@@ -0,0 +1,5 @@
+kind: FEATURES
+body: Add new endpoint that updates an agent's config for supported keys
```

Preface body entry with name of package/file `targets/bzero`

```suggestion
body: targets/bzero: Add new endpoint that updates an agent's config for supported keys
```